next.clayui.com/content/docs/components/label.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Label"
+description: "Labels are a visual pattern used to categorize information providing quick and easy recognition."
+packageNpm: "@clayui/label"
+packageVersion: "3.0.0"
+packageStatus: "Beta"
+---
+
+import {Label, LabelClosingActions, LabelDisplayTypes, LabelLink, LabelSizes, LabelWithChildren} from '../../../src/components/clay/Label';
+
+<div class="nav-toc">
+
+- [Variants and Best Pratices](#variants-and-best-pratices)
```

Comment:
I would say the same thing here that I mentioned in the button documentation. https://github.com/liferay/clay/pull/2062#discussion_r294529739